packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -444,6 +453,7 @@ const PipelineEditor = forwardRef(
     const [selectedNodeIDs, setSelectedNodeIDs] = useState<string[]>();
     const handleSelectionChange = useCallback((e: CanvasSelectionEvent) => {
       setSelectedNodeIDs(e.selectedNodes.map((n: NodeTypeDef) => n.id));
+      setPropertiesTab(e.selectedNodes.length > 0);
     }, []);
```

Comment:
you either need to either add `setPropertiesTab` as a dependency or move the logic or function here. I would just move the logic here since nothing else uses this function

Reply:
I'll move the logic, I had it in a function because I was calling it in a bunch of places, before I realized that calling it here covered every case.